.circleci/config.yml

```diff
@@ -145,13 +114,70 @@ workflows:
                 "no-warning-imports-guava",
                 "no-warning-imports-java-design-patterns",
                 "sonarqube",
+                "checkstyle-and-sevntu",
+                "spotbugs-and-pmd",
+                "site",
+                "release-dry-run",
+                "assembly-run-all-jar",
+                "no-error-test-sbe",
               ]
+              script-name: [ *validation_script ]
+              image-name: [ *custom_img ]
```

Comment:
Is this acting as a variable that is defined on 90?

Reply:
Yes